### ISSUE TITLE
ganeti-htools-0.3.0 requires pandoc during make

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#196]
Originally reported of Google Code with ID 140.

```
While I was packaging ganeti-htools-0.3.0 I encountered a problem where running make tried to call pandoc. This seems to be an error as the man pages are already included in the tarball.

Referenced thread: http://groups.google.com/group/ganeti/msg/77ad72c7a5f8b2fb
```
Originally added on 2011-02-11 17:42:32 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
The problem is due to how git-archive works:

  $ tar tzvf ganeti-htools-0.3.0-3-g55bdba0.tar.gz|grep man
  drwxrwxr-x root/root         0 2011-02-14 10:29 ganeti-htools-0.3.0-3-g55bdba0/man/
  -rw-rw-r-- root/root      3520 2011-02-14 10:29 ganeti-htools-0.3.0-3-g55bdba0/man/hail.rst
  -rw-rw-r-- root/root     27530 2011-02-14 10:29 ganeti-htools-0.3.0-3-g55bdba0/man/hbal.rst
  -rw-rw-r-- root/root      3253 2011-02-14 10:29 ganeti-htools-0.3.0-3-g55bdba0/man/hscan.rst
  -rw-rw-r-- root/root     14672 2011-02-14 10:29 ganeti-htools-0.3.0-3-g55bdba0/man/hspace.rst
  -rw-r----- root/root     29809 2011-02-10 14:44 ganeti-htools-0.3.0-3-g55bdba0/man/hbal.1
  -rw-r----- root/root      3567 2011-02-10 14:44 ganeti-htools-0.3.0-3-g55bdba0/man/hscan.1
  -rw-r----- root/root      3672 2011-02-10 14:44 ganeti-htools-0.3.0-3-g55bdba0/man/hail.1
  -rw-r----- root/root     15736 2011-02-10 14:44 ganeti-htools-0.3.0-3-g55bdba0/man/hspace.1

Feb 14th is when I generated the archive, whereas the man pages on disk were generated a while ago. Note that the on-disk RST files are also old, but when git-archive creates the archive, the timestamps are reset to “now”:

  $ ll man/
  total 176
  -rw-r----- 1 iustin eng  3672 Feb 10 14:44 hail.1
  -rw-r----- 1 iustin eng  4789 Feb 10 14:44 hail.html
  -rw-r----- 1 iustin eng  3520 Feb 10 11:04 hail.rst
  -rw-r----- 1 iustin eng 29809 Feb 10 14:44 hbal.1
  -rw-r----- 1 iustin eng 32429 Feb 10 14:44 hbal.html
  -rw-r----- 1 iustin eng 27530 Feb 10 13:43 hbal.rst
  -rw-r----- 1 iustin eng  3567 Feb 10 14:44 hscan.1
  -rw-r----- 1 iustin eng  4833 Feb 10 14:44 hscan.html
  -rw-r----- 1 iustin eng  3253 Feb 10 11:04 hscan.rst
  -rw-r----- 1 iustin eng 15736 Feb 10 14:44 hspace.1
  -rw-r----- 1 iustin eng 19650 Feb 10 14:44 hspace.html
  -rw-r----- 1 iustin eng 14672 Feb 10 11:04 hspace.rst

I'll see how to work around this.
```
Originally added on 2011-02-14 09:45:21 +0000 UTC.

Added Labels: Priority-Low Component-htools

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in commit d9f0298025, and will be released with 0.3.1 (which will probably be in a few weeks):

commit d9f0298
Author: Iustin Pop <iustin@google.com>
Date:   Mon Feb 14 11:47:28 2011 +0100

    Fix dist archive generation
    
    As reported in issue 140, the timestamps of the generated files can be
    older than the commit timestamp, leading to needless rebuilds of man
    pages and such. Furthermore, with the man pages not built, it
    currently fails, as there is no dependency on them.
    
    Therefore, we change the ‘dist’ rule so that it forces a
    maintainer-clean and then explicitly rebuilds the needed targets, so
    that we have up-to-date timestamps.
    
    Last, we also ship the HTML versions of the man pages.


thanks,
iustin
```
Originally added on 2011-02-17 15:23:04 +0000 UTC.

Mentioned Issue with GoogleCode ID 140 has been migrated to GitHub with ID #196.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:08 +0000 UTC.

Changed State: Released
